winpr/libwinpr/clipboard/synthetic_file.c

```diff
@@ -269,34 +247,35 @@ static WCHAR* concat_remote_name(const WCHAR* dir, const WCHAR* file)
 		return NULL;
 
 	memcpy(buffer, dir, len_dir * sizeof(WCHAR));
-	buffer[len_dir] = L'\\';
+	buffer[len_dir] = L'/';
```

Comment:
`Be careful with `L'somestring'`, windows understands `UCS-2/UTF16` most other systems `UCS4/UTF32`